metricbeat/docs/fields.asciidoc

```diff
@@ -46811,6 +46811,26 @@ type: keyword
 
 --
 
+*`kubernetes.pod.status.reason`*::
++
+--
+The reason the pod is in its current state (Evicted, NodeAffinity, NodeLost, Shutdown or UnexpectedAdmissionError)
+
+
+type: keyword
+
+--
+
+*`kubernetes.pod.status.ready_time`*::
++
+--
+Time in unix timestamp for a pod to achieve readiness 
```

Comment:
maybe would be better:
```suggestion
Time in unix timestamp when a pod achieved readiness 
```
or the same as in the ksm: `Readiness achieved time in unix timestamp for a pod`